radioactivedecay/inventory.py

```diff
@@ -828,6 +831,172 @@ def decay_modes(self) -> Dict[str, List[str]]:
             nuc: Nuclide(nuc, self.decay_data).decay_modes() for nuc in self.contents
         }
 
+    def calculate_decay_data(
+        self,
+        time_period: Union[float, np.ndarray],
+        time_units: str = "s",
+        time_scale: str = "linear",
+        decay_units: str = "Bq",
+        npoints: int = 501,
+    ) -> pd.DataFrame:
+        """
+        Returns a dataframe with the initial isotope and all decay progeny decayed for the amount of time specified by
+        time_period.
+
+        Parameters
+        ----------
+        time_period : Union[float, np.ndarray]
+            Time to decay the chain for. If a float is given, <time_scale> and <npoints> is used to create an evenly spaced array of numbers.
+            If an numpy ndarray is provided, the values contained within are used and <npoints> is ignored.
+        time_units : str, optional
+            Units for time series. Options are 'ps', 'ns', 'μs', 'us', 'ms', 's', 'm', 'h', 'd', 'y',
+            'ky', 'My', 'By', 'Gy', 'Ty', 'Py', and common spelling variations. Default is 's', i.e.
+            seconds. Use 'readable' to get strings of the half-lives in human-readable units.
```

Comment:
Delete `Use 'readable' to get strings of the half-lives in human-readable units.` as not used here 🙌